phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/ClasspathRepository.java

```diff
@@ -76,6 +79,11 @@ public RepositoryDTO fetchInstallableApplications() {
         }
     }
 
+    @Override
+    public boolean isSafe() {
```

Comment:
I'm not sure we need the concept of a safe repository at all. I see two possibilities:
1. You want to access Java code which is safe to use. In this case, that code should be provided as part of the API (safe Bean).
2. You want to access Java code which might be dangerous. We cannot allow that some user executes this script so you cannot be allowed to mark the repository as safe.